components/automate-ui/src/app/pages/+compliance/+reporting/reporting.component.ts

```diff
@@ -145,6 +145,14 @@ export class ReportingComponent implements OnInit, OnDestroy {
     }
   ];
 
+  private noValuesControlTagFilter = {
+    id: '',
+    score: 1,
+    text: 'no value',
+    title: 'no value',
+    version: ''
+  };
```

Comment:
It didn't look like we have a type for this anywhere.  Can anyone advise if its OK that i leave this open? 

This is the object built to hijack our suggestions array in order to display a "no value" option.

Reply:
I feel it's OK for now, given the scope of this PR. This area of the codebase is one of the oldest and at this point could use some more rework to make the code flow much simpler and easier to work with. Defining/utilizing more types, leveraging ngrx, and using facades along w/ other recently adopted patterns would most definitely benefit this area of the code but all that is probably best done w/ more planning and in separate PRs focused on those topics.